### DISCUSSION TITLE
自定义svg图标无法旋转

### DISCUSSION THREAD

**Discussion by pincman** [ant-design/ant-design#33346] [Q&A]
[这里](https://codesandbox.io/s/zi-ding-yi-tu-biao-antd-4-17-3-forked-06vr4?file=/index.js)自定义svg图标，用`Icon`组件包装后，传入`spin`属性没任何效果

**Comment by yangzheli** [comment]
https://github.com/ant-design/ant-design-icons/issues/270#issuecomment-671894456
